tools/clang/test/CodeGenDXIL/hlsl/types/matrix/selMat.hlsl

```diff
@@ -0,0 +1,25 @@
+// RUN: %dxc -E main -T ps_6_0 -HV 2018 %s | FileCheck %s
+// RUN: %dxc -E main -T ps_6_0 -HV 2018 -Zpr %s | FileCheck %s
+
+// CHECK-COUNT-16: fcmp fast olt float %{{.+}}, 3.000000e+00
+
+// CHECK-COUNT-16: select i1
+
+// CHECK: icmp ne i32
+// CHECK: 0
```

Comment:
This line is weird. What is it checking for? Similar lines in matElt were changed to be clearer.